handle_data/all_normalizations.py

```diff
@@ -180,6 +192,37 @@ def save_norms_to_file(self, js_file, missdir_ok: bool = True):
         with open(js_file, "w") as jsf:
             js.dump(norm_serialized, jsf)
 
+    @staticmethod
+    def extract_logit_epsilon(transform_str, default_eps=1e-6):
+        """
+        Extracts the epsilon value from a logit transformation string.
+
+        Examples of valid strings:
+            "logit_eps1e-06"
+            "logit_0.01"
+            "logit" (will return default_eps)
+        
+        :param transform_str: The transformation string to parse.
+        :param default_eps: Default epsilon value to return if no specific value is found.
+        :return: The extracted epsilon value or the default value.
+        """
+        if not transform_str.startswith("logit"):
+            raise ValueError(f"Invalid transformation string: {transform_str}. Expected to start with 'logit'.")
+
+        # Try to match 'logit_eps<value>' or 'logit_<value>'
+        match = re.match(r"logit(?:_eps|_)?([0-9.eE+-]+)?", transform_str)
```

Comment:
I will prefer using `r"logit(?:_eps|_)?(?P<val>[0-9.eE+-]+)?"` over `r"logit(?:_eps|_)?([0-9.eE+-]+)?` and `eps_str = match.groupdict()['val']`